helm/kiam/templates/server-daemonset.yaml

```diff
@@ -105,6 +105,11 @@ spec:
           {{- if .Values.server.extraEnv }}
 {{ toYaml .Values.server.extraEnv | indent 12 }}
         {{- end }}
+          {{- if .Values.server.seLinuxOptions }}
+          securityContext:
+            seLinuxOptions:
+          {{ toYaml .Values.server.seLinuxOptions | indent 14 }}
```

Comment:
The leading whitespace breaks this; the YAML fails to parse. I'll be creating a PR, but noting here in case anyone else is running into the issue